src/seedu/addressbook/ui/TextUi.java

```diff
@@ -113,7 +106,7 @@ public void showInitFailedMessage() {
     /** Shows message(s) to the user */
     public void showToUser(String... message) {
         for (String m : message) {
-            out.println(LINE_PREFIX + m.replace("\n", LS + LINE_PREFIX));
+            out.println(Formatter.beautifyEnds(m, LINE_PREFIX));
```

Comment:
How come the formatting of the showWelcomeMessage was not in the Formatter.java. Can improve SRP further I think